src/main/java/org/dswarm/graph/index/ChronicleMapUtils.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * This file is part of d:swarm graph extension.
+ *
+ * d:swarm graph extension is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * d:swarm graph extension is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with d:swarm graph extension.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.dswarm.graph.index;
+
+import java.io.File;
+import java.io.IOException;
+
+import net.openhft.chronicle.map.ChronicleMap;
+import net.openhft.chronicle.map.ChronicleMapBuilder;
+
+/**
+ * @author tgaengler
+ */
+public final class ChronicleMapUtils {
```

Comment:
No, I think that we don't use something from Chronicle right now at. So, yes we can remove this dependency for now.